templates/stringy_game.eex

```diff
@@ -24,7 +24,7 @@
                   <%= cell_presenter.mark(cell) %>
                 </div>
               <%= else %>
-                <form action="/tictactoe/moves/<%= cell_presenter.index(cell) %>" method="post" class="move-button">
+                <form action='/ttt/moves/<%= cell_presenter.index(cell) %>/<%= stringy_game_state %>' method="post" class="move-button">
```

Comment:
not sure on the `stringy_game_state` variable name - feels more verbose than it needs to be. I somehow feel just having `game_state` would be more clearer? I can see why you would name it `stringy_` to differentiate between it and the actual `game_state`, but I think in the view it's a safe assumption that the variables in the view are modified.